code/modules/customitems/item_spawning.dm

```diff
@@ -292,7 +295,7 @@ GLOBAL_LIST_EMPTY(character_id_to_custom_items_mapping)
 		target_mob.UpdateDamageIcon()
 		return
 
-	if(ispath(citem.item_path, /obj/item/organ/external))
+	if(body_only && ispath(citem.item_path, /obj/item/organ/external))
```

Comment:
I think this would be cleaner like:

```
if(body_only)
	if(ispath(whatever))
		whatever
	return
```

Insead of the double check with that return in the middle that would be hit after anyways?